src/mmw/js/src/app.js

```diff
@@ -67,6 +73,20 @@ var App = new Marionette.Application({
         }
     },
 
+    getLayersModel: function() {
+        return this.layers;
+    },
+
+    showLayerPicker: function() {
+        if (_.contains(settings.get('map_controls'), 'LayerSelector')) {
```

Comment:
I mean something like this:

(in settings.js)

```
function isLayerSelectorEnabled() {
    return _.contains(settings.get('map_controls'), 'LayerSelector');
}
```

Just to make this line of code somewhat self-documenting.